api.go

```diff
@@ -700,6 +711,40 @@ func (lapi *Client) CreateStream(csr CreateStreamReq) (*Stream, error) {
 	return stream, nil
 }
 
+func (lapi *Client) UpdateStream(id string, usr UpdateStreamReq) (*Stream, error) {
```

Comment:
Also curious on why do we need this function on record tester? Won't we create the stream with a policy already?

Reply:
Yeah not needed! Added it while working because I wanted to make it public after, but decided to discard the thing as cache would complicate it anyway. I left the function here for future use, but I can remove it with no problems